tests/test_router_bgp.rb

```diff
@@ -588,14 +614,46 @@ def test_default_confederation_peers
     bgp.destroy
   end
 
-  def test_set_get_log_neighbor_changes
-    skip(XR_SUPPORTED_BROKEN) if platform == :ios_xr
-    %w(test_default test_vrf).each do |t|
-      if t == 'test_default'
-        bgp = setup_default
+  def test_log_neighbor_changes_default
+    log_neighbor_changes(setup_default)
+  end
+
+  def test_log_neighbor_changes_vrf
+    log_neighbor_changes(setup_vrf)
+  end
+
+  def log_neighbor_changes(bgp)
+    if platform == :ios_xr
+      @device.cmd('term width 300')
+      # Lets assert based on XR standard output if a command is not found
+      pattern = 'No such configuration item'
+
+      if @vrf == 'default'
+        vrf_str = ''
       else
-        bgp = setup_vrf
+        vrf_str = "vrf #{@vrf}"
       end
+
+      # Test default
+      s = @device.cmd("show running-config router bgp #{@asnum} " \
+                      "#{vrf_str} bgp log neighbor changes disable")
+      assert_match(pattern, s,
+                   'bgp log_neighbor_changes disable should not be enabled')
+
+      # Disable log neighbors
+      bgp.log_neighbor_changes = false
+      s = @device.cmd("show running-config router bgp #{@asnum} " \
+                      "#{vrf_str} bgp log neighbor changes disable")
+      refute_match(pattern, s,
+                   'bgp log_neighbor_changes disable should be enabled')
+
+      # Enable log neighbors
+      bgp.log_neighbor_changes = true
+      s = @device.cmd("show running-config router bgp #{@asnum} " \
+                      "#{vrf_str} bgp log neighbor changes disable")
+      assert_match(pattern, s,
+                   'bgp log_neighbor_changes disable should not be enabled')
+    else
       bgp.log_neighbor_changes = true
       assert(bgp.log_neighbor_changes,
```

Comment:
same question here - for XR it looks like you're *only* testing the running-config, and not what the node_utils report, which seems wrong.